org.dupot.littleadventure.yml

```diff
@@ -0,0 +1,28 @@
+id: org.dupot.littleadventure
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: littleAdventure.x86_64
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+  - --filesystem=~/.local/share/godot/app_userdata/LittleAdventure
```

Comment:
This shouldn't be needed.

Reply:
I meant only the 11th line… x11, pulseaudio and dri should stay.

Reply:
I looked a bit at how bad XDG support is with Godot, and it seems that either you put `xdg-config/godot:create` and then the app as settings for all the other games, or you put this sepecificaly for the game.

Note that `--persist` also should work and is probably even better.